mo_net/samples/cbow.py

```
@@ -452,15 +501,38 @@ def train(
+@click.option(
+    "--temperature",
+    type=float,
+    default=1.0,
+    help="Temperature for sampling (higher = more random)",
+)
+@click.option(
+    "--top-p", type=float, default=0.9, help="Top-p (nucleus) sampling parameter"
 )
-def infer(model_path: Path):
-    """Start interactive REPL for word similarity queries"""
+def infer(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)